CDCgov/prime-simplereport
# Display test detail when clicking patient name on results pg (#3601)

# FRONTEND PULL REQUEST

## Related Issue
Fixes #3601 
This change helps make the "Test details" modal more discoverable.

## Changes Proposed
- On the "Results" tab, an org administrator should be able to click the patient's name and view the details of the test result in a modal.

## Additional Information
@kenieh had an open question in the issue about whether or not we want to remove the "View details" button in the Actions menu.

## Testing
1. Log in as an org admin.
2. Go to the "Results" page
3. Have at least one result
4. Click on the user's name and it should open the "Test details" modal
 
## Screenshots / Demos
<img width="1436" alt="Screen Shot 2022-05-13 at 4 07 10 PM" src="https://user-images.githubusercontent.com/20211771/168389850-e210dd8b-a4b3-468d-8042-8068ef31fc1a.png">
<img width="1436" alt="Screen Shot 2022-05-13 at 4 08 45 PM" src="https://user-images.githubusercontent.com/20211771/168389879-01f1531c-aff9-4665-86a2-4d9c2d01a933.png">

## Checklist for Author and Reviewer

### Design

### Content

### Support

### Security
